Examples/Io/Root/src/RootMaterialTrackWriter.cpp

```diff
@@ -211,14 +218,15 @@ FW::ProcessCode FW::RootMaterialTrackWriter::writeT(
         m_step_ez.push_back(posPos.z());
       }
 
+      // Store surface information
       if (m_cfg.storesurface) {
         const Acts::Surface* surface = mint.surface;
-        Acts::GeometryID layerID;
+        Acts::GeometryID slayerID;
```

Comment:
The renaming to `slayer` is done to make it clear to come from a surface?

Reply:
Yes since we have the same think for volume I though it would be clearer to have a slayerID for the surfaces and a vlayerID for the volumes.